src/librustc_lexer/src/tests.rs

```diff
@@ -128,6 +128,11 @@ fn check_lexing(src: &str, expect: Expect) {
     expect.assert_eq(&actual)
 }
 
+#[test]
+fn smoke_test() {
+    check_lexing("/* my source file */ fn main() { println!(\"zebra\"); }\n", expect![[]])
```

Comment:
>expect![[]]

The test output wasn't updated?

Reply:
Ah indeed... At least now we now that these tests are actually capable of failing on CI:

https://github.com/rust-lang/rust/runs/1044897833?check_suite_focus=true#step:23:2341